src/test/java/com/google/api/generator/spring/composer/goldens/EchoSpringAutoConfigurationGrpcRest.golden

```diff
@@ -95,8 +95,8 @@ public class EchoSpringAutoConfiguration {
    * default retry settings when they are not specified in EchoSpringProperties.
    */
   @Bean
-  @ConditionalOnMissingBean
-  public EchoClient echoClient(
+  @ConditionalOnMissingBean(name = "echoSettings")
```

Comment:
Adding comment to golden file for convenience:
```suggestion
  @ConditionalOnMissingBean
```
Reasoning in https://github.com/zhumin8/language_poc1/pull/5/files#r1039951844

Reply:
Removed `name = ...`